modules/ossm-security-mtls.adoc

```diff
@@ -99,3 +120,19 @@ The default is `TLS_AUTO` and does not specify a version of TLS.
 |`TLSv1_3`
 |TLS version 1.3
 |===
+
+[id="ossm-security-enabling-controlplane_{context}"]
+== Enabling strict mTLS for Mixer telemetry or policy components
+
+Secure connections are always used when proxies communicate with the control plane regardless of the `spec.security.controlPlane.mtls` setting. If you use Mixer telemetry or policy, set `spec.security.controlPlane.mtls` to `true` in your `ServiceMeshControlPlane` resource to enable strict mTLS.
+
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+spec:
+  version: v2.0
+  security:
+    controlPlane:
```

Comment:
Should we mention that `spec.security.controlPlane.mtls` is **_only_** used for Mixer and policy settings?

Reply:
I think that is what this says. I don't think it needs more emphasis.